ui_framework/src/components/code_editor/code_editor.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import AceEditor from 'react-ace';
+
+import { htmlIdGenerator, keyCodes } from '../../../services';
+
+export class KuiCodeEditor extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      hintInactive: false
+    };
+    this.idGenerator = htmlIdGenerator();
+  }
+
+  enableOverlay() {
```

Comment:
I did the renaming, but we cannot move the `focus` to it, since otherwise we would also focus the hint, when leaving the texteditor (`onBlurAce`) which we don't want.

Reply:
Small nit, but can we move it down so that it's next to `startEditing`?